### PR TITLE
Feat: Pair liqudity info history validator

### PR DESCRIPTION
**Pull request by tmrdlt**
- Implemented task to validate pair liqudity info history and deleting falsly entries

### CODE REVIEW

File: src/tasks/pair-liquidity-info-history-validator.service.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import { MdwClientService } from '../clients/mdw-client.service';
+import { PairLiquidityInfoHistoryService } from '../database/pair-liquidity-info-history.service';
+import { Injectable, Logger } from '@nestjs/common';
+import { uniq } from 'lodash';
+import { Cron } from '@nestjs/schedule';
+import { getClient } from '../lib/contracts';
+import { MicroBlockHash } from '../lib/utils';
+
+const EVERY_5_MINUTES_STARTING_AT_02_30 =
+  '30 2,7,12,17,22,27,32,37,42,47,52,57 * * * *';
```
Comment (kenodressel) [inline src/tasks/pair-liquidity-info-history-validator.service.ts]:
```suggestion
  '30 2-57/5 * * * *';
```

Comment (kenodressel) [reply src/tasks/pair-liquidity-info-history-validator.service.ts]: https://stackoverflow.com/questions/12786410/run-cron-job-every-n-minutes-plus-offset

File: src/tasks/pair-liquidity-info-history-validator.service.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    } catch (error) {
+      this.isValidationRunning = false;
+      this.logger.error(`Validation failed. ${error}`);
+    }
+  }
+
+  async validatePairLiquidityInfoHistory() {
+    this.logger.log(`Started validating pair liquidity info history.`);
+    // Get current height
+    const currentHeight = await (await getClient())[0].getHeight();
```
Comment (kenodressel) [inline src/tasks/pair-liquidity-info-history-validator.service.ts]: use `.then`

File: src/tasks/pair-liquidity-info-history-validator.service.ts (hunk 1 above)
Comment (tmrdlt) [reply src/tasks/pair-liquidity-info-history-validator.service.ts]: Nice one, good find!

**Review by kenodressel — APPROVED** [review]

**Pull request merged**